### ISSUE TITLE
Remove inflect from harvest

### ISSUE THREAD

**Issue by ssebro** [agco/harvest#39]
It makes our lives harder w/o much benefit.

**Comment by kristofsajdak** [comment]
Agreed, don't see the value add of declaring resources in the singular form to have them auto-pluralised by inflect.

**Comment by ssebro** [comment]
Oh - this wasn't done because we can turn off inflect by supplying an option to harvester.
